### PR TITLE
[Feature] Add file storage

### PR DESCRIPTION
**Pull request by Tanax-Xt**
### Description

Added the ability to download files (with a name change), download a file by its name, access a file by name (without downloading).

Example: 
```
http://127.0.0.1:8000/api/v1/uploads
http://127.0.0.1:8000/api/v1/uploads/*
http://127.0.0.1:8000/uploads/*.*
```

### Changes

* Added a method for uploading files
* Added a method for getting the file
* Added file browsing by name
* The formatting of the code was carried out

### Related Issues

Resolves task "Add file storage" in https://github.com/orgs/rapid-integration/projects/1

### CODE REVIEW

File: backend/.env [hunk 1]
```diff
@@ -10,3 +10,5 @@ POSTGRES_PASSWORD=root
 POSTGRES_PORT=5432
 POSTGRES_HOST=localhost
 POSTGRES_PATH=postgres
+
+FILES_PATH=../uploads
```
Comment (zobweyt) [inline backend/.env]: Rename it to `UPLOADS_PATH` to follow consistency

File: backend/src/core/config.py [hunk 2]
```diff
@@ -23,6 +23,8 @@ class Settings(BaseSettings):
     POSTGRES_HOST: str
     POSTGRES_PATH: str
 
+    FILES_PATH: str
```
Comment (zobweyt) [inline backend/src/core/config.py]: Rename it to `UPLOADS_PATH` to follow consistency

**Comment by zobweyt** [comment]
Could you please include `aiofiles` in `requirements/common.txt` file?

**Comment by Tanax-Xt** [comment]
Of course! `iofiles` has already been added in `requirements/common.txt` file

File: backend/src/core/files.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import secrets
+
+import aiofiles
+from fastapi import UploadFile
+from fastapi.responses import FileResponse
+
+from src.core.config import settings
+
+
+def generate_name(file: UploadFile):
```
Comment (zobweyt) [inline backend/src/core/files.py]: Rename it to `generate_unique_name` for clarity

File: backend/src/core/files.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    file_type = file.filename.split(".")[-1]
+    filename = f"{secrets.token_hex(nbytes=12)}.{file_type}"
+    while os.path.exists(os.path.join(settings.FILES_PATH, filename)):
+        filename = f"{secrets.token_hex(nbytes=12)}.{file_type}"
```
Comment (zobweyt) [inline backend/src/core/files.py]: Avoid copy-pasting `f"{secrets.token_hex(nbytes=12)}.{file_type}"`. Extract it to a function like `generate_name`

**Review by zobweyt — CHANGES_REQUESTED** [review]
LGTM

File: backend/.env (hunk 1 above)
Comment (Tanax-Xt) [reply backend/.env]: Done

File: backend/src/core/config.py (hunk 2 above)
Comment (Tanax-Xt) [reply backend/src/core/config.py]: Done

File: backend/src/core/files.py (hunk 3 above)
Comment (Tanax-Xt) [reply backend/src/core/files.py]: Done

File: backend/src/core/files.py (hunk 4 above)
Comment (Tanax-Xt) [reply backend/src/core/files.py]: Done

**Review by zobweyt — APPROVED** [review]

**Pull request merged**